xbmc/pvr/PVRManager.h

```diff
@@ -564,7 +564,13 @@ namespace PVR
     /*!
      * @brief Query the events available for CEventStream
      */
-    CEventStream<ManagerState>& Events() { return m_events; }
+    CEventStream<PVREvent>& Events() { return m_events; }
```

Comment:
@ksooo CEventStream does not guarantee that events are delivered in the same order as sent. not sure if this can cause issues here.